test/services/xccdf_report_parser_test.rb

```diff
@@ -220,17 +221,17 @@ class TestParser < ::XccdfReportParser
         benchmark: @report_parser.benchmark
       ).save(validate: false)
 
+      assert_equal 367, @report_parser.op_rules.count
       assert_difference('Rule.count', 365) do
         @report_parser.save_rules
       end
 
-      @report_parser.rules = nil
-
+      # All rules already exist so none are saved
```

Comment:
Setting `@report_parser.rules` will obviously make `save_rules` save 0 rules, so let's make the test a bit more difficult by removing it, so that we check that the parser does not import the same rules twice.

Reply:
That's a good point - I actually changed it at a point when I removed the `||=` cache, I reverted that change but not this test chnage